app/views/posts/index.html.erb

```diff
@@ -1 +1,13 @@
-<h1>List of all posts for a single users</h1>
+<section class="showView">
+  <ul class="postsList">
+   <%= render 'shared/user_card', user: @user %>
+      <% @posts.each do |p| %>
+        <%= link_to (user_post_path(p.author, p)) do %>
+          <li>
+            <%= render 'shared/post_card', post: p, max: 5 %>
+          </li>
+        <% end %>
+      <%end%>
+  </ul>
```

Comment:
**Changes Required**
- [ ] Good effort with using `max` to return a limit of 5 comments. We should consider using a method from the models that returns the `most recent 5 comments` i.e something like `@post.recent_comments`

Reply:
Done! added reference to `take_five_recent` method in partial  `_post_card`!